client/my-sites/plans/jetpack-plans/faq/index.tsx

```diff
@@ -146,7 +146,7 @@ const JetpackFAQ: FC = () => {
 							className="jetpack-faq__section"
 						>
 							{ translate(
-								'Anyone with at least five websites can join our licensing platform and enjoy a 25% discount across all Jetpack products! You can learn more about our {{agenciesLink}}licensing platform and agency program here{{/agenciesLink}}.',
+								'Anyone with at least five websites can join our licensing platform and enjoy up to 60% discount across all Jetpack products! You can learn more about our {{agenciesLink}}licensing platform and agency program here{{/agenciesLink}}.',
```

Comment:
I'd recommend refactoring this slightly to make the 60% an argument instead of having it right in the string again. That way, whenever it changes again, it won't trigger a string re-translation. You can see an example of this here: fbhepr%2Skers%2Spnylcfb%2Spyvrag%2Szl%2Qfvgrf%2Spurpxbhg%2Spbzcbfvgr%2Qpurpxbhg%2Spbzcbaragf%2Snqqvgvbany%2Qgrezf%2Qbs%2Qfreivpr%2Qva%2Qpneg.gfk%3Se%3Qr9q2s791%23246-og